main.go

```diff
@@ -123,7 +123,12 @@ func main() {
 	api.GET("/version", getVersionHandler)
 	api.GET("/swagger/*any", ginSwagger.WrapHandler(swaggerFiles.Handler))
 
-	_ = r.Run()
+	var bindAddress string
+	var bindAddressDefined bool
+	if bindAddress, bindAddressDefined = os.LookupEnv("BIND_ADDRESS"); !bindAddressDefined {
+		bindAddress = "0.0.0.0:8080"
+	}
+	_ = r.Run(bindAddress)
```

Comment:
Gin does read port by default from environment variable `PORT` https://github.com/gin-gonic/gin/blob/34ce2104cad324f444943c528746bf6d23643cd3/utils.go#L139-L153

(it's not well documented in though)

So we're only gaining address here. But it's a nice-to-have feature for those that need it though.

Reply:
Cool didnt know about the port binding. This will let me bind the adress to the docker vlan though which is a plus.